docs/shared/getting-started/intro.md

```diff
@@ -1,118 +1,85 @@
 
-{% link-card title="TypeScript Monorepo" type="tutorial" url="/getting-started/tutorials/typescript-packages-tutorial" icon="jsMono" /%}
+```json
+{
+  "devDependencies": {
+    "nx": "latest"
+  }
+}
+```
 
-{% link-card title="React Monorepo" type="tutorial" url="/getting-started/tutorials/react-monorepo-tutorial" icon="reactMono" /%}
+Now you can run:
 
-{% link-card title="Angular Monorepo" type="tutorial" url="/getting-started/tutorials/angular-monorepo-tutorial" icon="angularMono" /%}
+```shell
+nx build my-project
+```
 
-{% link-card title="Gradle Monorepo" type="tutorial" url="/getting-started/tutorials/gradle-tutorial" icon="gradle" /%}
+This will execute the `build` script from `my-project`'s `package.json`, equivalent to running `npm run build` in that project directory.
 
-{% /cards %}
+From there, you can gradually enhance your setup by adding features like [task caching](/features/cache-task-results), [task distribution](/ci/features/distribute-task-execution), and other powerful capabilities as your needs grow.
 
-## Pick Your Stack!
+{% /callout %}
 
-{% cards cols="3" lgCols="8" mdCols="6" smCols="5" moreLink="/showcase/example-repos" %}
+Nx Core works great alone, but you can incrementally add capabilities as needed:
 
-{% link-card title="Express" appearance="small" url="/technologies/node/express" icon="express" /%}
-{% link-card title="Vue" appearance="small" url="/technologies/vue/introduction" icon="vue" /%}
-{% link-card title="Next" appearance="small" url="/technologies/react/next" icon="nextjs" /%}
-{% link-card title="Nuxt" appearance="small" url="/technologies/vue/nuxt/introduction" icon="nuxt" /%}
-{% link-card title="Nest" appearance="small" url="/technologies/node/nest" icon="nestjs" /%}
-{% link-card title="Remix" appearance="small" url="/technologies/react/remix" icon="remix" /%}
-{% link-card title="Expo" appearance="small" url="/technologies/react/expo" icon="expo" /%}
-{% link-card title="React Native" appearance="small" url="/technologies/react/react-native" icon="react" /%}
-{% link-card title="Fastify" appearance="small" url="/showcase/example-repos/mongo-fastify" icon="fastify" /%}
-{% link-card title="Svelte" appearance="small" url="/showcase/example-repos/add-svelte" icon="svelte" /%}
-{% link-card title="Solid" appearance="small" url="/showcase/example-repos/add-solid" icon="solid" /%}
-{% link-card title="Lit" appearance="small" url="/showcase/example-repos/add-lit" icon="lit" /%}
-{% link-card title="Astro" appearance="small" url="/showcase/example-repos/add-astro" icon="astro" /%}
-{% link-card title="Qwik" appearance="small" url="/showcase/example-repos/add-qwik" icon="qwik" /%}
+- [**Nx Cloud**](/ci): Scales builds through [remote caching](/ci/features/remote-cache), [distributed task execution](/ci/features/distribute-task-execution), [automated task splitting](/ci/features/split-e2e-tasks), and intelligent flaky task detection. Ensures CI performance scales linearly with codebase growth.
```

Comment:
```suggestion
- [**Nx Cloud**](/ci): Scales builds through [remote caching](/ci/features/remote-cache), [distributed task execution](/ci/features/distribute-task-execution), [automated task splitting](/ci/features/split-e2e-tasks), and intelligent flaky task detection. Ensures CI performance scales with codebase growth.
```

idk if we should claim linearity

Reply:
no that was a mistake that got thrown in by AI when updating the sentence 😅. Linear scaling is what happens without Nx haha. Already fixed that locally on my branch